arwen/cryptoapi/cryptoei.go

```diff
@@ -197,10 +224,12 @@ func v1_4_keccak256(context unsafe.Pointer, dataOffset int32, length int32, resu
 	runtime := arwen.GetRuntimeContext(context)
 	crypto := arwen.GetCryptoContext(context)
 	metering := arwen.GetMeteringContext(context)
+	initialGasLeft := metering.GasLeft()
 
 	memLoadGas := math.MulUint64(metering.GasSchedule().BaseOperationCost.DataCopyPerByte, uint64(length))
 	gasToUse := math.AddUint64(metering.GasSchedule().CryptoAPICost.Keccak256, memLoadGas)
 	metering.UseGas(gasToUse)
+	runtime.TraceGasUsed(keccak256Name, initialGasLeft)
```

Comment:
it would be pretty easy to do runtime.TraceGasUsed(keccak256name, gasToUse) instead of inital gas left and computation